src/client.py

```diff
@@ -0,0 +1,32 @@
+"""Something client-y."""
+
+import sys
+import socket
+
+
+def client(message):
+
+    message += 'EOF'
+    print('message to send is: ' + message)
+    infos = socket.getaddrinfo('127.0.0.1', 5001)
+    stream_info = [i for i in infos if i[1] == socket.SOCK_STREAM][0]
+    client = socket.socket(*stream_info[:3])
+    client.connect(stream_info[-1])
+
+    client.sendall(message.encode('utf8'))
```

Comment:
In Python 2 you won't need to `encode` here because `message` will already be a bytestring.